### PR TITLE
OPS-1423 Update Portfolio Title Card

### PR DESCRIPTION
**Pull request by stevtek**
## What changed

Portfolio Title Card updated for design changes 
* Update the Team Leader tag to match recent Figma design
* Add the label "Portfolio Description" above the description text

## Issue

#1423 

## How to test

* view the portfolio details page for a portfolio and verify the title card design

## Screenshots
image

## Definition of Done Checklist
- [x] OESA: Code refactored for clarity
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- ~[ ] Form validations updated~

## Links

* Figma designs here: https://www.figma.com/file/dsQFe6G2KjcTWNq7NNZujm/OPRE-OPS?type=design&node-id=5001-159899&mode=dev

### CODE REVIEW

File: frontend/src/components/UI/TeamLeaders/TeamLeaders.jsx [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
+                    ) : (
+                        <dd className="margin-0 margin-top-1 margin-bottom-2">
+                            <Tag
+                                tagStyle="primaryDarkTextLightBackground"
+                                text="TBD"
+                            />
+                        </dd>
+                    )}
+                </dl>
+                {/*<h3 className={`font-sans-2xs padding-right-2`}>Team Leaders </h3>*/}
```
Comment (fpigeonjr) [inline frontend/src/components/UI/TeamLeaders/TeamLeaders.jsx:32]: commented code here

**Review by fpigeonjr — APPROVED** [review]
💪 looks good Steve

File: frontend/src/components/UI/TeamLeaders/TeamLeaders.jsx (hunk 1 above)
Comment (stevtek) [reply frontend/src/components/UI/TeamLeaders/TeamLeaders.jsx]: thx, I've removed it

**Review by tdonaworth — APPROVED** [review]

**Review by johndeange — APPROVED** [review]

**Comment by andressacurvelo** [comment]
The design looks good! Thank you!

**Pull request merged**